lib/redis_dedupe/version.rb

```diff
@@ -1,3 +1,6 @@
+# frozen_string_literal: true
+
+# :nodoc:
 module RedisDedupe
-  VERSION = "0.0.3"
+  VERSION = "0.0.4"
```

Comment:
This is one thing I was wondering about myself. Since the behavior is to re-run previously failed blocks, which could cause the loop to switch from "finishable" to "unfinishable" if the execution stoppage is repeatable, is it a major version bump perhaps? Seems like a potentially breaking change, but also, the behavior now does seem like a bug as well. Thoughts?